packages/nova-react/src/graphql/hooks.test.tsx

```diff
@@ -152,9 +156,87 @@ describe(useFragment, () => {
       useFragment(fragment, opaqueFragmentRef);
     void _;
   });
+
+  it("allows null or undefined to be passed as the fragment ref", () => {
+    const fragment = {} as unknown as GraphQLTaggedNode;
+
+    const { result, rerender } = renderHook<
+      { fragmentRef: null | undefined },
+      null | undefined
+    >(({ fragmentRef }) => useFragment(fragment, fragmentRef), {
+      wrapper: ({ children }) => (
+        <NovaGraphQLProvider graphql={{}}>{children}</NovaGraphQLProvider>
+      ),
+      initialProps: {
+        fragmentRef: null,
+      },
+    });
+
+    expect(result.current).toBeNull();
+
+    rerender({ fragmentRef: undefined });
+
+    expect(result.current).toBeUndefined();
+  });
+
+  it("return type can be `null`, `undefined` or data", () => {
```

Comment:
Nit: "it return type …" isn't English

Reply:
True, but technically in the jest report it turns into `useFragment return type ...` ;)

Reply:
Will fix though as it doesn't read a nice here :)